R/dvhydrograph-render.R

```diff
@@ -89,6 +97,11 @@ createDVHydrographPlot <- function(reportObject){
   estimated3EdgesDf <- as.data.frame(estimated3Edges, stringsAsFactors = FALSE)
   estimated4EdgesDf <- as.data.frame(estimated4Edges, stringsAsFactors = FALSE)
   comparisonEdgesDf <- as.data.frame(comparisonEdges, stringsAsFactors = FALSE)
+  
+  #remove zeros from field measurements, if present:
+  if (excludeZeroNegativeFlag && !isEmptyOrBlank(fieldVisitMeasurements)) {
+    fieldVisitMeasurements <- removeZeroNegative(fieldVisitMeasurements)
+  }
```

Comment:
I think you only need to check `isLogged` in the if statement since `excludeZeroNegativeFlag` and `!isEmptyOrBlank(fieldVisitMeasurements)` were checked in order to create `logAxis`.